zeebe/snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/FileBasedSnapshotChunkReader.java

```diff
@@ -118,16 +149,8 @@ private ByteBuffer encodeChunkId(final CharSequence path) {
   }
 
   private CharSequence decodeChunkId(final ByteBuffer id) {
-    return chunkIdView.wrap(id);
-  }
-
-  private static final class CharSequenceView {
-    private final DirectBuffer wrapper = new UnsafeBuffer();
-    private final AsciiSequenceView view = new AsciiSequenceView();
-
-    private CharSequence wrap(final ByteBuffer buffer) {
-      wrapper.wrap(buffer);
-      return view.wrap(wrapper, 0, wrapper.capacity());
-    }
+    final DirectBuffer buffer = new UnsafeBuffer(id);
+    final var chunkId = new AsciiSequenceView().wrap(buffer, 0, buffer.capacity());
+    return chunkId.toString().split("-")[0];
```

Comment:
:x: It was like this before but I think this code is not quite correct. It does not take `ID_CHARSET` into account. I think the following would work correctly:
```suggestion
    return ID_CHARSET.decode(id).toString().split("-")[0];
```